lib/manifique/web_client.rb

```diff
@@ -130,14 +130,22 @@ def parse_mask_icon_from_html
       if mask_icon_link = @html.at_css("link[rel=mask-icon]")
         icon = { "purpose" => "mask-icon" }
         icon["src"] = mask_icon_link.attributes["href"].value rescue nil
-        return if icon["src"].to_s.empty?
+        return unless is_adequate_src(icon["src"])
         icon["type"]  = link.attributes["type"].value rescue get_icon_type(icon["src"])
         icon["color"] = mask_icon_link.attributes["color"].value rescue nil
         @metadata.icons.push icon
         @metadata.from_html.add "icons"
       end
     end
 
+    def is_data_url?(src)
+      !!src.match(/^data:/)
+    end
+
+    def is_adequate_src(src)
+      !src.nil? && !is_data_url?(src)
```

Comment:
What if the `src` is just an empty string?

Reply:
Hmm, yeah. I just reverted to the old implementation for checking the presence, so it catches that, too.